terraform/platforms/aws/variables.tf

```diff
@@ -41,6 +41,10 @@ variable "vpc_id" {
   description = "ID of the VPC where the subnets are defined."
 }
 
+variable "route53_prefix" {
+  description = "Defines the primary dns prefix for the Route53 entry"
```

Comment:
would you mind keeping "etcd" as the default value here, so that we do not break existing deployments?

Reply:
Of course. Done !!